tests/e2e/simplekvtest.go

```diff
@@ -490,3 +504,79 @@ func KVRewardsTest(log *logging.Logger, conn *grpc.ClientConn, rtc client.Runtim
 
 	return nil
 }
+
+// KVTxGenTest1min generates random transactions for 1 minute.
+func KVTxGenTest1min(log *logging.Logger, conn *grpc.ClientConn, rtc client.RuntimeClient) error {
+	ctx := context.Background()
+	ac := accounts.NewV1(rtc)
+
+	log.Info("getting Alice's account balance")
+	ab, err := ac.Balances(ctx, client.RoundLatest, testing.Alice.Address)
+	if err != nil {
+		return err
+	}
+	var balance uint64
+	if q, ok := ab.Balances[types.NativeDenomination]; ok {
+		// We can do this only because the account's balance fits into an uint64.
+		balance = q.ToBigInt().Uint64()
+	} else {
+		return fmt.Errorf("Alice's account is missing native denomination balance")
+	}
+
+	numAccounts := 10
+	coinsPerAccount := uint64(200)
+
+	minBalanceRequired := coinsPerAccount * uint64(numAccounts)
+	if balance < minBalanceRequired {
+		return fmt.Errorf("Alice is too broke to fund accounts (balance is %d, need %d)", balance, minBalanceRequired)
+	}
+
+	// Create RNG.
+	seed := time.Now().UnixNano()
+	rngSrc, err := drbg.New(crypto.SHA512, []byte(fmt.Sprintf("%d%d%d%d", seed, seed, seed, seed)), nil, []byte("KVTxGenTest1min"))
+	if err != nil {
+		return err
+	}
+	rng := rand.New(mathrand.New(rngSrc))
+
+	// Generate accounts.
+	log.Info("generating accounts", "num_accounts", numAccounts, "rng_seed", seed)
+	var accts []signature.Signer
+	numT := make(map[string]uint64)
+	for i := 0; i < numAccounts; i++ {
+		// Create account.
+		at := txgen.AccountType(uint8(rng.Intn(int(txgen.AccountTypeMax) + 1)))
+		numT[at.String()]++
+		sig, err := txgen.CreateAndFundAccount(ctx, rtc, testing.Alice.Signer, i, at, coinsPerAccount)
+		if err != nil {
+			return err
+		}
+
+		accts = append(accts, sig)
+	}
+	log.Info("accounts generated", "num_accts_per_type", numT)
+
+	// Generate random transactions for 1 minute.
+	log.Info("generating transactions")
+	txgenCtx, cancel := context.WithTimeout(ctx, 60*time.Second)
+	defer cancel()
+
+	// Generate a new random tx every 250ms until txgenCtx timeouts.
+	gens := append([]txgen.GenerateTx{}, txgen.DefaultTxGenerators...)
+	gens = append(gens, DefaultKVTxGenerators...)
+	err, genErrs, subErrs, ok := txgen.Generate(txgenCtx, rtc, rng, accts, gens, 250*time.Millisecond)
+	if err != nil {
+		return err
+	}
+
+	if ok == 0 {
+		return fmt.Errorf("no generated transactions were submitted successfully")
+	}
+
+	// Note that submission errors are fine here, since we're going to get
+	// invalid nonce errors a lot, because the txs are generated in parallel.
+	// Transaction generation errors are also fine, since queries can fail
+	// due to yet nonexisting keys in the keyvalue storage, etc.
```

Comment:
so this is more to test a network's stability?

Reply:
From what I understand, that was the goal, yes :)
Later, we could make a separate test using the txgen that does more intelligent insert/get/remove operations that should always succeed, so that it can fail whenever an error happens.

Reply:
Filed as #152 :)